src/components/Header.js

```diff
@@ -82,20 +86,39 @@ class HeaderContainer extends Component {
     } else {
       alert('Sorry your browser doesn\'t support the Geolocation API');
     }
-  }
 
+  }
+  
   render () {
+    const that = this;
+    if (!navigator.online) {
+      db.weather.get(1, function(obj) {
+        that.setState({currentWeather: obj.temp})
+        }).catch(function(error){
+        console.log(error)
+      })
+
+      db.city.get(2, function(obj) {
+        that.setState({currentCity: obj.city})
+        console.log("state: " + that.state.currentCity)
+      }).catch(function(error){
+        console.log(error)
+      })
+    }
+
     const {currentCity, currentWeather} = this.state;
     const message = (this.state.currentCity && this.state.currentWeather)
-      ? ` You're in ${currentCity}. It is currently ${currentWeather}°F`
-      : ``;
-
+    ? `You're in ${currentCity}. It is currently ${currentWeather}°F`
+    : 
+    `It is currently ${currentWeather}°F. You're in ${currentCity}.`;
```

Comment:
Is this just to let us know it's working?

Reply:
Oh yea, I had that as a debugging to know when it was loading offline stuff. I can remove it to clean it up.